frontend/bonsai_app/blueprints/sample/templates/resistance_variants.html

```diff
@@ -321,5 +323,11 @@ <h5>Variant calling result</h5>
         window.localStorage.setItem("selectedVariants", variantsObj)
     }
     // reset selected varants on page load
+    const selectAllVariants = (elem, tableId) => {
```

Comment:
Have you considered separating out the code into a separate JS file? To keep things organized.

I am also pondering whether you would like to move the "selected" state into some kind of data structure, such as an array, rather than relying on the UI itself. I.e. update the data structure and then propagate the state of that one into the UI components. I don't think you need or should change that right here. But could be something to ponder the impact of over all here. We can discuss at some point!

Reply:
That is a good suggestion.